### PR TITLE
fix(artifact-test): verify write_back_cache is never set

### PR DESCRIPTION
**Pull request by soyacz**
write_back_cache is considered bogus and should never be set (see https://github.com/scylladb/seastar/issues/1008).
When https://github.com/scylladb/scylla-pkg/issues/3099 is fixed we should start veryfing this setting is not set at all (like on AWS).

Should be merged when https://github.com/scylladb/scylla-pkg/issues/3099 is fixed.
refs: https://github.com/scylladb/qa-tasks/issues/435

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — APPROVED** [review]
LGTM

**Comment by fgelcer** [comment]
don't merge it until https://github.com/scylladb/scylla-pkg/issues/3099 will be fixed.

another thing is, once this one will be merged, i think we will not be able to FF to branch-5.1 anymore.

**Comment by yaronkaikov** [comment]
@soyacz @fgelcer we are not using `write_back_cache` in `gce` or `azure` - https://github.com/scylladb/scylla-machine-image/blob/054945af3f9974feb1c078caa40e703fb094f013/packer/scylla_install_image#L120

**Review by roydahan — APPROVED** [review]

**Comment by yaronkaikov** [comment]
@roydahan @fgelcer @soyacz please see @avikivity comment - https://github.com/scylladb/scylla-machine-image/pull/394#discussion_r968298643

**Review by roydahan — CHANGES_REQUESTED** [review]
Get requirements from Avi or whoever decide about that

**Review by vladzcloudius — APPROVED** [review]
LGTM

**Comment by vladzcloudius** [comment]
> @roydahan @fgelcer @soyacz please see @avikivity comment - [scylladb/scylla-machine-image#394 (comment)](https://github.com/scylladb/scylla-machine-image/pull/394#discussion_r968298643)

Please, see my response to that comment, @yaronkaikov.

**Review by fgelcer — CHANGES_REQUESTED** [review]
need to get a final answer from Avi, and then the scylla-pkg PR merged, then we will come back to this one. for now, it needs to wait for @yaronkaikov, and once he approves it (after his PR is merged), we need to merge this one as well

**Comment by fruch** [comment]
seem like there is now decision on https://github.com/scylladb/scylla-machine-image/pull/394,

will reopen once and if there gonna be a decision

**Pull request closed without merging**
